source/JABHandler.py

```diff
@@ -656,6 +656,24 @@ def getAccessibleTextRange(self, start, end):
 		bridgeDll.getAccessibleTextRange(self.vmID, self.accContext, start, end, buf, length)
 		return textUtils.getTextFromRawBytes(buf.raw, numChars=length, encoding=textUtils.WCHAR_ENCODING)
 
+	# Constant gotten from AccessBridgePackages.h,
+	# minus one to accommodate the null character
+	MAX_BUFFER_SIZE = 10239
+
+	def getAccessibleTextRange(self, start, end):
+		length = (end + 1) - start
+		if length <= 0:
+			return ""
+
+		text = []
+		while start <= end:
+			bufferSize = min(self.MAX_BUFFER_SIZE, length)
+			text.append(self._javaGetAccessibleTextRange(start, start + bufferSize - 1))
+			start += bufferSize
+			length = (end + 1) - start
```

Comment:
Can we just reduce by `bufferSize`?
```suggestion
			length -= bufferSize
```